must-gather/Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM registry.access.redhat.com/ubi8/go-toolset:1.14.7 as gobuilder
+FROM registry.access.redhat.com/ubi8/go-toolset:1.17.10 as gobuilder
 
-RUN go get github.com/google/pprof
+RUN export GO111MODULE=on && go install -v github.com/google/pprof@latest
```

Comment:
Why do we need this export?

Reply:
https://github.com/openshift/oadp-operator/blob/e4f87247e56bc78530b9c78554b5175806d00a95/must-gather/README.md#support-for-insecure-tls-connections

Reply:
Otherwise cannot install `github.com/google/pprof`.I tried a bunch of things and this was the first one that worked.

Reply:
"since the default behavior in Go 1.16 is now GO111MODULE=on, this trick isn’t necessary anymore"
Maybe the first thing that work is the same to not specifying anything

Reply:
Updated. Thanks, Tiger